index.phtml

```diff
@@ -53,7 +53,7 @@ include ("externals/menu.phtml");
 <?php // FIXME: I18N printf, markup: ?>
         <?=T_("Looking for someone to contact about <span class=\"gnucash\">GnuCash</span>? Try the")?>
           <a href="https://wiki.gnucash.org/wiki/Mailing_Lists"><?=T_("Mailing Lists")?></a>
-          <?=T_(", which are public, email-based discussion groups where you can ask and answer <span class=\"gnucash\">GnuCash</span> questions to other users.")?>
+          <?=T_(", which are public, e-mail-based discussion groups where you can ask and answer <span class=\"gnucash\">GnuCash</span> questions to other users.")?>
```

Comment:
"email" is valid, and is the predominant form used in the English-speaking Gnucash community.